templates/cc/const.go

```diff
@@ -2,8 +2,8 @@ package tpl
 
 const constTpl = `{{ $r := .Rules }}
 	{{ if $r.Const }}
-		if {{ accessor . }} != {{ lit $r.GetConst }} {
-			return {{ err . "value must equal " $r.GetConst }}
+		if ({{ accessor . }} != {{ lit $r.GetConst }}) {
+			{{ err . "value must equal " $r.GetConst }}
```

Comment:
Prefer that the generated code matches Google/Envoy style, so 2 space indents.

Reply:
The generated code is kind of a mess as far as formatting goes (see output code gist [here](https://gist.github.com/akonradi/33f7fe85de29edce2fad5653ffffb6f0)) so I would rather invest time later in either cleaning it up or (preferably) using an auto-formatter. To be fair, the generated Go code is also a mess, and could benefit greatly from formatting as well.